src/mono/mono/metadata/icall-eventpipe.c

```diff
@@ -151,6 +151,32 @@ eventpipe_execute_rundown (
 	ep_rt_mono_fire_assembly_rundown_events_func assembly_events_func,
 	ep_rt_mono_fire_method_rundown_events_func methods_events_func);
 
+static
+gboolean
+eventpipe_walk_managed_stack_for_thread_func (
```

Comment:
Are these declarations needed here ?

Reply:
Followed convention in this file to forward declare statics.